src/main/java/com/ericsson/ei/config/CheckEIConfigurations.java

```diff
@@ -0,0 +1,95 @@
+                + "waitlist.fixedRateResend: " + env.getProperty("waitlist.fixedRateResend") + "\n"
+                + "subscription.collection.name: " + env.getProperty("subscription.collection.name") + "\n"
+                + "subscription.collection.repeatFlagHandlerName: " + env.getProperty("subscription.collection.repeatFlagHandlerName") + "\n"
+                + "testaggregated.enabled: " + env.getProperty("testaggregated.enabled") + "\n"
+                + "threads.corePoolSize: " + env.getProperty("threads.corePoolSize") + "\n"
+                + "threads.queueCapacity: " + env.getProperty("threads.queueCapacity") + "\n"
+                + "threads.maxPoolSize: " + env.getProperty("threads.maxPoolSize") + "\n"
+                + "missedNotificationCollectionName: " + env.getProperty("missedNotificationCollectionName") + "\n"
+                + "missedNotificationDataBaseName: " + env.getProperty("missedNotificationDataBaseName") + "\n"
+                + "email.sender: " + env.getProperty("email.sender") + "\n"
+                + "email.subject: " + env.getProperty("email.subject") + "\n"
+                + "notification.failAttempt: " + env.getProperty("notification.failAttempt") + "\n"
+                + "notification.ttl.value: " + env.getProperty("notification.ttl.value") + "\n"
+                + "spring.mail.host: " + env.getProperty("spring.mail.host") + "\n"
+                + "spring.mail.port: " + env.getProperty("spring.mail.port") + "\n"
+                + "spring.mail.username: " + env.getProperty("spring.mail.username") + "\n"
+                + "spring.mail.properties.mail.smtp.auth: " + env.getProperty("spring.mail.properties.mail.smtp.auth") + "\n"
+                + "spring.mail.properties.mail.smtp.starttls.enable: " + env.getProperty("spring.mail.properties.mail.smtp.starttls.enable") + "\n"
+                + "er.url: " + env.getProperty("er.url") + "\n"
+                + "ldap.enabled: " + env.getProperty("ldap.enabled") + "\n"
+                + "ldap.url: " + env.getProperty("ldap.url") + "\n"
+                + "ldap.base.dn: " + env.getProperty("ldap.base.dn") + "\n"
+                + "ldap.username: " + env.getProperty("ldap.username") + "\n"
+                + "ldap.user.filter: " + env.getProperty("ldap.user.filter") + "\n"
+                + "logging.level.root: " + env.getProperty("logging.level.root") + "\n"
+                + "logging.level.org.springframework.web: " + env.getProperty("logging.level.org.springframework.web") + "\n"
+                + "logging.level.com.ericsson.ei: " + env.getProperty("logging.level.com.ericsson.ei") + "\n"
+                + "\nThese properties is only some of the configuraitons, more configurations can have been provided.\n");
+        
+        try {
+            new RulesHandler().readRuleFileContent(rulePath);
+        } catch (IOException e) {
+            LOGGER.debug("Rules file can't be loaded/read. RuleFile path: " +  rulePath, e.getMessage(), e);
```

Comment:
Maybe catch the FileNotFoundException separately, and not print out the full stacktrace for this error. It should be enough with: "java.io.FileNotFoundException: File 'RULE PATH' does not exist"

Reply:
It can be other issues as well,, e.g. Chmod issues,,, can maybe catch the general Exception class "Exception e".

Reply:
Maybe I wasn't clear in my first comment. What I meant was that the FileNotFound exception is probably a very common issue in which we can know right away what caused it and don't have to log the entire stacktrace. It's enough with just the error message.  For other exceptions, I agree with logging the stacktrace can help find out the root cause of the problem.

Reply:
"e" exception has already been removed from the log message. Look in the Files Changed page/tab.

Reply:
True, wasn't looking at the latest changes...